public/locales/es/translation.json

```
@@ -1752,7 +1753,13 @@
     "noPledges": "No se encontraron promesas",
     "noCampaigns": "No se encontraron campañas",
     "raised": "Recaudado",
-    "amountRaised": "Monto Recaudado"
+    "amountRaised": "Monto Recaudado",
+    "campaignName": "Nombre de la campaña",
+    "startDate": "Fecha de inicio",
```

Biome (2.1.2)

[error] The key searchCampaigns was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)